app/src/main/java/campus/tech/kakao/map/model/DBHelper.kt

```diff
@@ -0,0 +1,77 @@
+import android.content.ContentValues
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import campus.tech.kakao.map.model.Place
+
+class DBHelper(context: Context) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
+
+    /**
+     *  코틀린에서 사용하는 companion object는 클래스 내부에 포함된 정적 멤버 변수와 메서드를 선언할 때 사용
+     */
+    companion object {
+        private const val DATABASE_NAME = "place.db"
+        private const val DATABASE_VERSION = 1
+
+        private const val TABLE_NAME = "place"
+        private const val COLUMN_ID = "id"
+        private const val COLUMN_NAME = "name"
+        private const val COLUMN_LOCATION = "location"
+    }
+
+    // 테이블 생성 등 초기 데이터베이스 설정
+    override fun onCreate(db: SQLiteDatabase) {
+        val createTableQuery = """
+            CREATE TABLE $TABLE_NAME (
+                $COLUMN_ID INTEGER PRIMARY KEY AUTOINCREMENT,
+                $COLUMN_NAME TEXT NOT NULL,
+                $COLUMN_LOCATION TEXT NOT NULL
+            )
+        """
+        db.execSQL(createTableQuery)
+    }
+
+    // 데이터베이스 버전이 변경될 때 실행되는 업그레이드 로직
+    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
+        db.execSQL("DROP TABLE IF EXISTS $TABLE_NAME")
+        onCreate(db)
+    }
+
+    // 데이터 추가
+    fun insertData(name: String, location: String) {
+        val db = this.writableDatabase
+        val values = ContentValues()
+        values.put(COLUMN_NAME, name)
+        values.put(COLUMN_LOCATION, location)
```

Comment:
이부분은 조금도 코틀린스러운 방식으로 apply를 이용해 이렇게도 정의해볼 수 있어요.
```suggestion
        val values = ContentValues().apply {
                put(COLUMN_NAME, name)
                put(COLUMN_LOCATION, location)
        }
```
🔗 [코틀린의 apply, with, let, also, run](https://medium.com/@limgyumin/%EC%BD%94%ED%8B%80%EB%A6%B0-%EC%9D%98-apply-with-let-also-run-%EC%9D%80-%EC%96%B8%EC%A0%9C-%EC%82%AC%EC%9A%A9%ED%95%98%EB%8A%94%EA%B0%80-4a517292df29)